tests/foreman/api/test_computeresource_vmware.py

```diff
@@ -98,6 +109,7 @@ def test_positive_provision_end_to_end(
                     'controller_key': 1001,
                 },
             },
+            'virtual_tpm': 'false' if pxe_loader.vm_firmware == 'bios' else 'true',
```

Comment:
This should work, but in the hammer PR we mentioned that the value should be 0 or 1 https://github.com/theforeman/hammer-cli-foreman/pull/630

Reply:
yes, both works, but I think both makes sense, so do we really need to change it?